libs/renterd-types/src/bus.ts

```diff
@@ -320,7 +320,7 @@ export type ContractsAddParams = {
 export type ContractsAddPayload = {
   contract: ContractRevision
   startHeight: number
-  totalCost: Currency
+  initialRenterFunds: Currency
```

Comment:
@peterjan was this field also renamed in the contracts add and renew endpoint payloads?